clang/bindings/python/clang/cindex.py

```diff
@@ -2663,6 +2671,21 @@ def visitor(field, children):
         conf.lib.clang_Type_visitFields(self, fields_visit_callback(visitor), fields)
         return iter(fields)
 
+    def get_bases(self):
+        """Return an iterator for accessing the base classes of this type."""
+
+        def visitor(field, children):
+            assert field != conf.lib.clang_getNullCursor()
+
+            # Create reference to TU so it isn't GC'd before Cursor.
+            field._tu = self._tu
+            fields.append(field)
+            return 1  # continue
+
+        fields: list[Cursor] = []
+        conf.lib.clang_visitCXXBaseClasses(self, fields_visit_callback(visitor), fields)
```

Comment:
Since it is used for different purposes as well now, it might be sensible to change the name of `fields_visit_callback` to something more generic.

Reply:
Any suggestions for what to call it?  I'm drawing a blank.

Reply:
Hm something like `base_visit_callback`?
But I also just noticed that the paramter in `clang_visitCXXBaseClasses` is of type `CXFieldVisitor`, so I guess the current name is fair after all... up to you I would say!